src/marlin_stubs/G26.hpp

```diff
@@ -0,0 +1,165 @@
+// G26.hpp
+
+#pragma once
+
+/// Path of Manhattan snake
+/// Alternate X and Y coordinates.
+/// It cannot print diagonals etc.
+/// First X and Y are a starting point.
+static const constexpr float snake1[] = {
```

Comment:
this data structure is very hard to read and verify (even though optimal for the CPU)
ideally if you could at least distinguish x and y ... something like `float [][2] = { { x, y }, {x, y} ...`